mscore/scoreaccessibility.cpp

```diff
@@ -228,6 +229,7 @@ std::pair<int, float> ScoreAccessibility::barbeat(Element *e)
       int ticks = 0;
       TimeSigMap* tsm = e->score()->sigmap();
       Element* p = e;
+      int ticksB = ticks_beat(tsm->timesig(0).timesig().denominator());
```

Comment:
This doesn't take into account the fact that time signature may change during the score: by calling `tsm->timesig(0)` you obtain the time signature event for the beginning of the score (tick = 0) but that time signature may be different from the one that is actual during the current tick. You should probably pass the correct current tick to `timesig()` function, and it probably makes sense to do so only for the case of segment parent element (see below in the code) as other cases yield 0 beat value anyway.

Reply:
Yes I forgot to consider the change of time signature. Thank you!
One question here: what kind of element has a measure parent instead of a segment?

Reply:
There are a few - measure numbers, repeat texts like "DS al Coda", probably others.